unit_converter/__init__.py

```diff
@@ -24,19 +24,18 @@
 from urllib.error import URLError
 from urllib.request import urlopen
 
-import albert
 import inflect
 import pint
+from albert import *
 
-
-md_iid = '2.0'
+md_iid = "2.1"
 md_version = "1.4"
```

Comment:
please update the version if you change the extension.

Reply:
Good catch, fixed in 1d34d579a1ae8f56f954e201d61f5c021131de55